test/controllers/api/v2/hosts_controller_test.rb

```diff
@@ -724,9 +724,8 @@ def test_rebuild_tftp_config
       Setting[:restrict_registered_smart_proxies] = false
       SETTINGS[:require_ssl] = false
 
-      Resolv.any_instance.stubs(:getnames).returns(['else.where'])
```

Comment:
This was very confusing as it is supposed to not get called, so why stubbing it? :)